css/footer.css

```diff
@@ -0,0 +1,34 @@
+.footer_container h3 {
+  margin-top: 2rem;
+}
+
+.footer_container {
+  text-align: center;
+  margin-top: 3rem;
+  border-top: 1px solid grey;
```

Comment:
you can use variables for these colors as well

Reply:
yup.Will do that.